src/scheduler.cpp

```diff
@@ -182,7 +182,7 @@ void SingleThreadedSchedulerClient::AddToProcessQueue(std::function<void()> func
     MaybeScheduleProcessQueue();
 }
 
-void SingleThreadedSchedulerClient::EmptyQueue()
+void SerialTaskRunner::clear()
```

Comment:
In commit "scripted-diff: Rename SingleThreadedSchedulerClient to SerialTaskRunner" (46defde7a60afe186629481b738225919f874342)

I think I made a bad suggestion earlier to call this method "clear". I mistakenly thought this was discarding callbacks, but it appears to wait for all the callbacks to run. In light of this, I think it would be better to call this method "flush" or "wait".